jib-core/src/main/java/com/google/cloud/tools/jib/cache/json/CacheMetadataLayerPropertiesObjectTemplate.java

```diff
@@ -31,27 +31,28 @@
   /** Represents a pair of source files and extraction path. */
   public static class LayerEntryTemplate implements JsonTemplate {
 
-    /** The paths to the source files that the layer was constructed from. */
-    @Nullable private List<String> sourceFiles;
+    /** The path to the source file for this layer entry. */
+    @Nullable private String sourceFile;
 
-    /** The intended path to extract the source files to in the container. */
+    /** The intended path to extract the source file to in the container. */
     @Nullable private String extractionPath;
 
     @Nullable
-    public List<String> getSourceFiles() {
-      return sourceFiles;
+    public String getSourceFileString() {
+      return sourceFile;
     }
 
     @Nullable
-    public String getExtractionPath() {
+    public String getExtractionPathString() {
```

Comment:
I'm fine without the `String` suffix. The type says it. But up to you.

Reply:
I'm mostly trying to keep the method naming for the `LayerEntry` the same as this template object - so `getExtractionPath` would be the path form and `getExtractionPathString` would be the string form.